### ISSUE TITLE
Meaning of seed

### ISSUE THREAD

**Issue by asthaverma13** [yxgeee/MMT#11]
Hi,

As you have mentioned in your documentation 

sh scripts/pretrain.sh dukemtmc market1501 resnet50 1
sh scripts/pretrain.sh dukemtmc market1501 resnet50 2

1 and 2 are the seeds. What is the significance of seed in the code?

Waiting for reply

Thanks

**Comment by yxgeee** [comment]
We adopt different seeds for pre-training the model twice, and we can achieve two different initializations as a result. Loading different initialization weights for the coupled networks in our MMT framework could introduce more independent/complementary soft pseudo labels for training.
